packages/protocol/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  ra:
+    build: .
+    container_name: onchain-ra
+    environment:
+      - FORK_URL=${FORK_URL}
+      - PRIVATE_KEY=${PRIVATE_KEY}
+      - QUOTE=${QUOTE}
+    entrypoint: [ "/bin/bash", "-c", "PRIVATE_KEY=${PRIVATE_KEY} ./script/config_dcap_sgx_verifier.sh --quote ${QUOTE}"]
```

Comment:
so this is only for testing the sgx verifier? If so, shall we create a sub folder for this docker-compose file? Keeping it in the project's root folder doesn't make sense to me.